contracts/src/libraries/Predeploys.sol

```diff
@@ -19,6 +19,9 @@ library Predeploys {
     /// @dev We maximize compatibility with the contracts deployed by ZeframLou
     address internal constant Create3 = 0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf;
 
+    /// @notice ERC6551Registry address
```

Comment:
Please add the reason we are not using a namespace address (0xCCCC...), for example a link to the ERC.

Reply:
Added comments.